app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -306,6 +309,44 @@ def raw_connect(opts = {})
       true
     end
 
+    #
+    # Checks the metrics connection details.
+    #
+    # @api private
+    #
+    def check_connect_metrics(opts = {})
+      # Get the options and assign defaults:
+      username = opts[:metrics_username]
+      password = opts[:metrics_password]
+      server = opts[:metrics_server]
+      port = opts[:metrics_port] || 5432
+      database = opts[:metrics_database] || default_history_database_name
+
+      # Metrics are optional, so we should only check the details if the server has been specified:
+      return true unless server
+
+      # Decrypt the password:
+      password = MiqPassword.try_decrypt(password)
+
+      # Build the options that will be used to call the methods that checks that the metrics connection can
+      # be created:
+      opts = {
+        :username => username,
+        :password => password,
+        :host     => server,
+        :port     => port,
+        :database => database
+      }
+      begin
+        OvirtMetrics.connect(opts)
+        OvirtMetrics.connected?
+      rescue StandardError => error
+        raise adapt_metrics_error(error)
+      ensure
+        OvirtMetrics.disconnect rescue nil
```

Comment:
seems that rubocop would prefer 

        begin
          OvirtMetrics.disconnect
        rescue
          nil
        end

but we have the same style in line 145

Reply:
I will do as Rubocop says.